### DISCUSSION TITLE
Why is the demo not using the :hover pseudo-class?

### DISCUSSION THREAD

**Discussion by Frozenlock** [day8/re-com#269] [Q&A] [answered]
More a question than a real issue, but why is the demo going through all the trouble of keeping track of the mouse to update the background color of the left navigation panel, when it could just use CSS?

Is it just to demonstrate `re-com` capabilities?

**Comment by mike-thompson-day8** [comment, marked as answer]
When it comes to CSS we've been listening  to Facebook & Co:

https://speakerdeck.com/vjeux/react-css-in-js
http://formidablelabs.com/blog/2015/03/01/launching-radium/

I'm not saying we're living the dream just yet (we're actually still straddling the fence), but that's the direction we are nudging towards.

**Comment by Frozenlock** [comment]
I see, thanks!
